app/models/vm_or_template.rb

```diff
@@ -1224,21 +1224,6 @@ def self.parse_path(path)
     return storage_id, location
   end
 
-  # TODO: Vmware specific
-  # Finds a Vm by a full path of the Storage and location
-  def self.lookup_by_path(path)
-    begin
-      storage_id, location = parse_path(path)
```

Comment:
Is parse_path still called now?

Reply:
I think if parse_path can be removed, then the methods that parse_path calls(repository_parse_path, and location2uri) would only be left with callers in the vmware plugin and could be moved there directly:

https://github.com/ManageIQ/manageiq-providers-vmware/blob/bd06b1dec7bbb24f4603ef5799001e4b146461b4/app/models/manageiq/providers/vmware/infra_manager/inventory/parser/virtual_machine.rb#L63-L65

Reply:
Based on [this search](https://github.com/search?q=org%3AManageIQ+parse_path&type=Code) it looks like `parse_path` can be removed as well.

Reply:
the only caller of `repository_parse_path` is [the VMware provider](https://github.com/search?q=org%3AManageIQ+repository_parse_path&type=Code) so will move that in a future PR.

Reply:
[same](https://github.com/search?q=org%3AManageIQ+location2uri&type=Code) for `location2uri`

Reply:
and [no callers](https://github.com/search?q=org%3AManageIQ+uri2location&type=Code) of `uri2location`.  So, removing that as well